src/components/plots/xyPlot.ts

```diff
@@ -48,6 +39,12 @@ module Plottable {
       if (this._autoAdjustYScaleDomain) {
         this._updateYExtentsAndAutodomain();
       }
+
+      // TODO: is this needed?
+      if (xScale != null) {
+        xScale.onUpdate(this._adjustYDomainOnChangeFromXCallback);
+      }
```

Comment:
It's a question, left if there to remember.
@jtlan Is this needed? It used to be in the constructor, but not in the projector, which makes me thing it was a bug